src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2179,6 +2180,15 @@ namespace UnitarySystems {
                 this->m_IdleVolumeAirRate = this->m_MaxNoCoolHeatAirVolFlow;
                 this->m_IdleMassFlowRate = this->MaxNoCoolHeatAirMassFlow;
                 this->m_IdleSpeedRatio = this->m_IdleVolumeAirRate / this->m_DesignFanVolFlowRate;
+            } else {
+                for ( Iter = this->m_NumOfSpeedCooling; Iter > 0; --Iter ) {
+                    this->m_CoolVolumeFlowRate[ Iter ] = this->m_MaxCoolAirVolFlow * Iter / this->m_NumOfSpeedCooling;
```

Comment:
More allow 2-speed dx to be variable speed (ea021269050ac068e7def63e2093a434b3ed7aba)

Reply:
and this goes with the 2-speed coil addition to this section.